Ryujinx.Tests/Cpu/CpuTestAlu32.cs

```diff
@@ -77,6 +77,25 @@ private static uint[] _Lsr_Lsl_Asr_Ror_()
             CompareAgainstUnicorn();
         }
 
+        [Test, Pairwise]
+        public void Shadd8([Values(0u, 0xdu)] uint rd,
+                           [Values(1u)] uint rm,
+                           [Values(2u)] uint rn,
+                           [Random(RndCnt)] uint w0,
+                           [Random(RndCnt)] uint w1,
+                           [Random(RndCnt)] uint w2)
+        {
+            uint opcode = 0xE6300F90u; //SHADD8 R0, R0, R0
```

Comment:
```suggestion
            uint opcode = 0xE6300F90u; // SHADD8 R0, R0, R0
```
Sorry, missed that one.